dataland-frontend/tests/e2e/specs/company-upload-and-search/CompanyUpload.ts

```diff
@@ -14,7 +14,9 @@ describeIf(
       const companyName = "Test company XX";
       cy.visitAndCheckAppMount("/companies/upload");
       uploadCompanyViaForm(companyName).then((company) => {
+        cy.wait(5000);
         cy.visitAndCheckAppMount(`/companies/${company.companyId}`);
+        cy.wait(5000);
```

Comment:
are those wait statements for debug, or should they stay?

Reply:
This should stay, as it seemingly did the trick for the CD tests to be successful.

Reply:
5s? can it be shorter?

Reply:
Maybe, i will try 1s

Reply:
OK